docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build: .
     container_name: cms-docker
     environment:
-      - DATABASE_URL=postgresql://postgres:postgres@db:5432/cms?schema=public
+      - DATABASE_URL=postgresql://neondb_owner:y4sFwGKqeR3A@ep-white-wildflower-a53dtrhs.us-east-2.aws.neon.tech/neondb?sslmode=require
```

Comment:
so you decided to give us your db url

Reply:
Oops, that was a slip-up on my part! I am gonna revoke the credentials and fixe the issue. Won't happen again—thanks for catching it

Reply:
its not about revoking. restore to what it was